download_and_create_reference_datasets/v02/hail_scripts/write_dataset_validation_ht.py

```diff
@@ -0,0 +1,53 @@
+import argparse as ap
+import logging
+
+import hail as hl
+from hail_scripts.v02.utils.computed_fields.vep import get_expr_for_vep_sorted_transcript_consequences_array, \
+    get_expr_for_worst_transcript_consequence_annotations_struct, CONSEQUENCE_TERM_RANK_LOOKUP
+
+logging.basicConfig(format='%(asctime)s %(levelname)-8s %(message)s')
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+p = ap.ArgumentParser()
+p.add_argument("-g", "--genome-version", help="Genome build: 37 or 38", choices=["37", "38"], default="37")
+args = p.parse_args()
+
+
+def read_gnomad_subset(genome_version):
+    logger.info("==> Read gnomAD subset")
+
+    # select a subset of gnomad genomes variants that are in > 90% of samples
+    ht = hl.split_multi(hl.read_table('gs://gnomad-public/release/2.1.1/ht/genomes/gnomad.genomes.r2.1.1.sites.ht'))
```

Comment:
Multi-allelic variants are already split in the gnomAD 2.1+ Hail table and VCF.

See "Split all the multi-allelic variants" in https://macarthurlab.org/2018/10/17/gnomad-v2-1/

Reply:
Since this Hail table is based on GRCh37, should `read_gnomad_subset` raise an exception if GRCh38 is requested?

Reply:
Good to know about splitting, I didn't want to assume.
So there are a lot of cases not considered in this script (38 being one of them). The way I figured it, it's a script that's gonna be run probably once and most likely by me. So this is more of a good it done so we can make good progress.

Reply:
Also, when 38 is ready the path would probably include a 37 and 38, right? I'll change it when it happens.